priv/repo/migrations/20160203035956_create_character.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Yggdrasil.Repo.Migrations.CreateCharacter do
+  use Ecto.Migration
+
+  def change do
+    create table(:characters) do
+      add :name, :string
+      add :user_id, references(:users, on_delete: :nothing)
+      add :game_id, references(:games, on_delete: :nothing)
+
+      timestamps
+    end
+    create index(:characters, [:user_id])
+    create index(:characters, [:game_id])
+    create unique_index(:characters, [:name, :game_id])
```

Comment:
@copenhas we may want to include user_id if we wanted to limit a user to having only one character per game. Thoughts? I'm not too concerned with it at the moment, I figured at least those two fields should be good enough.